docs/src/pages/components/masonry/ImageMasonry.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function ImageMasonry() {
+  return (
+    <Masonry
+      columns={{ xs: 3, sm: 5, md: 7 }}
+      spacing={{ xs: 1, sm: 2, md: 3 }}
```

Comment:
I'n not sure spacing breakpoints are useful here, but if they need to be demonstrated, perhaps limit to 1 & 1.5?